functions/_fzf_configure_bindings_help.fish

```diff
@@ -1,43 +1,43 @@
+    fzf_configure_bindings installs key bindings for fzf.fish's features and erases any bindings it
+    previously installed. It installs bindings for both default and insert modes. fzf.fish executes
+    it without options on fish startup to install the out-of-the-box key bindings.
+
+    By default, feature are bound to a mnemonic key sequence, shown below. Each feature's binding
+    can be configured using a namesake corresponding option:
+        FEATURE            |  DEFAULT KEY SEQUENCE         |  CORRESPONDING OPTION
         Search directory   |  Ctrl+Alt+F (F for file)      |  --directory
         Search git log     |  Ctrl+Alt+L (L for log)       |  --git_log
         Search git status  |  Ctrl+Alt+S (S for status)    |  --git_status
         Search history     |  Ctrl+R     (R for reverse)   |  --history
         Search variables   |  Ctrl+V     (V for variable)  |  --variables
         Search processes   |  Ctrl+Alt+P (P for process)   |  --processes
-    An option with a key sequence value overrides the binding for its feature, while an option
-    without a value disables the binding. A feature that is not customized retains its default
-    menomonic binding specified above. Key bindings are installed for default and insert modes.
+    Override a feature's binding by specifying its corresponding option with the desired key
+    sequence. Disable a feature's binding by specifying its corresponding option with no value.
 
-    In terms of validation, fzf_configure_bindings fails if passed unknown options. Furthermore, it
-    expects an equals sign between an option's name and value. However, it does not validate key
-    sequences. Rather, consider using fish_key_reader to manually validate them.
+    Because fzf_configure_bindings erases bindings it previously installed, it can be cleanly
+    executed multiple times. Once the desired fzf_configure_bindings command has been found, add it
+    to your config.fish in order to persist the customized bindings.
 
-    In terms of experimentation, fzf_configure_bindings erases any bindings it previously installed
-    before installing new ones so it can be repeatedly executed in the same fish session without
-    problem. Once the desired fzf_configure_bindings command has been found, add it to config.fish
-    in order to persist the bindings.
+    In terms of validation, fzf_configure_bindings fails if passed unknown options. It expects an
+    equals sign between an option's name and value. However, it does not validate key sequences.
 
-    The -h and --help options print this help message.
+    Pass -h or --help to print this help message and exit.
 
 EXAMPLES
-    Install the default mnemonic bindings
-        \$ fzf_configure_bindings
-    Install the default bindings but override git log's binding to Ctrl+G
-        \$ fzf_configure_bindings --git_log=\cg
-    Install the default bindings but leave search history unbound
-        \$ fzf_configure_bindings --history
```

Comment:
One notable change is I think `--feature=` is a better convention than `--feature` to disable a feature, so I removed this example and updated the usage to reflect that.